docs/es/week06/06-2.md

```diff
@@ -0,0 +1,330 @@
+---
+lang-ref: ch.06-2
```

Comment:
`lang: es` missing.

Reply:
Sorry, i forgot that. I resolve that on this commit. [ES] Fix 06-2.md